appcli/orchestrators.py

```diff
@@ -577,6 +586,74 @@ def __exec_command(self, command: Iterable[str]) -> CompletedProcess:
         return subprocess.run(command, capture_output=True)
 
 
+class NullOrchestrator(Orchestrator):
```

Comment:
It'd be worth adding a quick section to the README about how to use the orchestrator.